### ISSUE TITLE
Bridge interface and MAC filtering issue

### ISSUE THREAD

**Issue by oblique** [joukewitteveen/netctl#111]
Hi,

When a bridge interface is created it always have the smallest MAC address of the slave interfaces.
If a user wants to have a bridge interface but he/she is connected to a network that has MAC filtering then he/she will not be able to establish a connection if the MAC address is not recognized by the router.

I'm willing to implement `MACAddressOf=<interface name>` which it will change the MAC address of bridge interface to the MAC address of `<interface name>` exactly after all slave interfaces are added.

What do you think?

**Comment by oblique** [comment]
I fixed my problem with the following:
```
Description="Bridge eth0-tap0"
Interface=br0
Connection=bridge
BindsToInterfaces=(eth0 tap0)
IP=no
ExecUpPost="ip link set dev br0 address $(cat /sys/class/net/eth0/address); IP=dhcp; ip_set"
ExecDownPre="IP=dhcp"

## Ignore (R)STP and immediately activate the bridge
SkipForwardingDelay=yes
```
But `MACAddressOf=<interface name>` would be a good feature to have.

**Comment by joukewitteveen** [comment]
sounds reasonable, please suggest an implementation.

**Comment by joukewitteveen** [comment]
This has also been brought up here: https://bugs.archlinux.org/task/64766

**Comment by joukewitteveen** [comment]
The requested functionality is available in netctl 1.21 via the `MACAddress=` option implemented in b5335f4350015d7c8b0cc66349d1290b7251bd9d. Note that the option also accepts interface names.
